src/main/java/org/apache/lucene/analysis/kr/utils/DictionaryUtil.java

```diff
@@ -312,12 +296,12 @@ public static String combineAndEomiCheck(char s, String eomi) throws MorphExcept
 	 * @param type	1: josa, 2: eomi
 	 * @throws MorphException
```

Comment:
The plugin depends on elasticsearch (2.3), which depends on guava 1.8. That is how guava jar is being added to the classpath at compile time. I don't see any change required here to fix the guava reference.